package.json

```diff
@@ -9,7 +9,8 @@
   },
   "scripts": {
     "contentlayer:build": "npx contentlayer build --clearCache",
-    "test": "yarn contentlayer:build"
+    "test": "yarn contentlayer:build",
+    "prettier": "npx prettier -w developers"
```

Comment:
This npm script you added for prettier would only work if prettier is installed globally by the user running it. Which is less than ideal. 

Can you add `prettier` it as a dev dependency? Eventually, it will be actually be used within a GitHub action anyway.

Reply:
`npx` is for local (`node_modules/.bin`) but yep re: `devDependency`. Will wait on what you think re the dead code though before spending any time on this.

Reply:
(and yep was thinking same re: adding an action)